users/user-guide-query/how-to-handle-unique-counting.md

```diff
@@ -26,11 +26,33 @@ Functions:
 
 * **DistinctCountHLL**(x)\_ -> LONG\_
 
-For column type **INT**/**LONG**/**FLOAT**/**DOUBLE**/**STRING** , Pinot treats each value as an individual entry to add into HyperLogLog Object, then compute the approximation by calling method **cardinality()**.
+For column type **INT**/**LONG**/**FLOAT**/**DOUBLE**/**STRING** , Pinot treats each value as an individual entry to add into HyperLogLog Object, then compute the approximation by calling method _**cardinality()**.
 
 For column type **BYTES**, Pinot treats each value as a serialized HyperLogLog Object with pre-aggregated values inside. The bytes value is generated by `org.apache.pinot.core.common.ObjectSerDeUtils.HYPER_LOG_LOG_SER_DE.serialize(hyperLogLog)`.
 
-All deserialized HyperLogLog object will be merged into one then calling method _\*\*cardinality() \*\*to get the approximated unique count._
+All deserialized HyperLogLog object will be merged into one then calling method _**cardinality()** to get the approximated unique count._
```

Comment:
to verify, can there be multiple object here, or is Object a proper term in the context of HyperLogLog? (see it's capitalized in line 29)

Reply:
would rephrasing as follows be accurate?:

All deserialized HyperLogLog objects are merged into one, and then Pinot calls the _**cardinality()** method to get the approximated unique count.

Reply:
i am ok with both this line or the original statement. actually i only made a small change to fix the format of this line.